### ISSUE TITLE
Program won't start - Java player doesn't open

### ISSUE THREAD

**Issue by Leed2822** [rpgboss/rpgboss#546]
After changing some sound (cursor) effects, every time I open the project now the terminal only opens and doesn't continue on after _EventDispatchThread.run (unknown source)_ 

Sound effects were mp3 and removing them did not work.

**Comment by Leed2822** [comment]
Here's a screenshot
![untitled](https://user-images.githubusercontent.com/17233378/49031289-8b94ff00-f177-11e8-872c-765e0c5b4b7e.png)

**Comment by EgyLynx** [comment]
"After changing some sound (cursor) effects"
Can that be?!"

**Comment by AntumDeluge** [comment]
I really don't know for sure, but the traceback reports a string index out of range. It's looking in a string at index 51. But the string probably doesn't have that many characters, thus the exception. It looks like it may be related to the filename of one (or some) of the MP3 files: "fixMp3Filename". This error could be because you removed the MP3s, but they are still referenced in the project. Do you know if the traceback message was different before you removed the MP3s?

**Comment by DEATHHAWX** [comment]
Yes it is because of using custom assets after i tried putting my own stuff a few days later i got that same error. but after deleting the project with those new assets then it would launch

**Comment by Leed2822** [comment]
@AntumDeluge 
I'm assuming that a traceback message is the code from the terminal program. And if so, .EventDispatchThread error did not repeatedly appear before I added/removed the custom cursor sounds. As of removing the sounds, I had only moved them out of the project folder. If removing them out of project reference (as you had mentioned in your response) is a solution to this issue, is it a possibility?  

Thanks

**Comment by DEATHHAWX** [comment]
when i had this i went through every file trying to delete anything that had my custom assets or had something to do with them. the problem would not get fixed unless i deleted the entire game through file manager and there was no way to recover. i also believe that even you delete the assets out of the folder they still remain in the rpgmap folder and i think thats what also keeps the error

**Comment by AntumDeluge** [comment]
@Leed2822 I think you would have to move/copy the items back into the project directories (if you still have them). Then you would have to open your project in RPGBoss & remove them from any maps & the game database. Then save your project & close RPGBoss. Then you should be able to delete/move them.

If you don't have the items anymore, you should be able to take any file of the same type, rename it to what the original item was, then put it in the project directories. For example: If you originally had an audio file named "cat.ogg" but don't have it anymore. Just download (or make) an .ogg music file from anywhere on the internet, rename it to "cat.ogg" then put it in your project directory.

**Comment by DEATHHAWX** [comment]
It is impossible to open the program if the custom assets are present if you copy. Delete open the program it will launch. Then paste them back in either nothing refreshes or it crashes and your back to square one.
